homeassistant/components/meater/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Meater Temperature Probe",
```

Comment:
This is a product name and doesn't need to be translated.

```suggestion
```

Reply:
Fixed in 143f8b4